### ISSUE TITLE
[SCI Feedback] Software Sustainability Actions

### ISSUE THREAD

**Issue by seanmcilroy29** [Green-Software-Foundation/software_carbon_intensity#302]
**Software Sustainability Actions**  submitted by Ben Logan [UBS]

All actions that can reduce the carbon emissions of a piece of software fall into one of three categories. The SCI specification intends to encourage more of these actions to be taken in developing software applications.

- Energy Efficiency: Actions taken to make software use less electricity to perform the same function.
- Hardware Efficiency: Actions taken to make software use less physical resources to perform the same function.
- Carbon Awareness: Actions taken to time or region-shift software computation to take advantage of clean, renewable or low carbon sources of electricity.
 
On first read, I found the lines on ‘Energy Efficiency’ and ‘Hardware Efficiency’ a little difficult to interpret. Reading it a couple of times, or a bit more carefully, it makes sense, but I wonder if it could be improved?

Perhaps this would be a clearer distinction;

- Software Efficiency: Actions taken to change software, such that it uses less energy to perform the same function and derive the same output. E.g. the use of a more efficient search algorithm
- Hardware Efficiency: Actions taken to deploy software in such a way that it uses less physical resources to perform the same function. E.g. the use of a modern CPU that requires less power

**Comment by Henry-WattTime** [comment]
I think these proposed definitions add clarity and should be integrated (with a few minor tweaks).

**Comment by atg-abhishek** [comment]
Agreed, makes sense to me as well - can supersede my comment in the PR on the terminology change.

**Comment by Henry-WattTime** [comment]
WG: Change to FAQ
